src/components/DayPicker.jsx

```diff
@@ -68,11 +72,17 @@ const defaultProps = {
   monthFormat: 'MMMM YYYY',
 };
 
+const triggerInitialVisibleMonth = (props) => (
+  props.hidden ? moment() : props.initialVisibleMonth()
```

Comment:
Can we pull this as an instance method onto the class, named as `getInitialVisibleMonth` or something? That way we just have access to the props straight up.

Actually we don't even call this more than once. Why don't we just have this logic directly in the constructor?